habitat/tests.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+#
+# Tests for the Chef-DK habitat package.
+# Assume package has already been installed.
+#
+
+set -eo pipefail
+
+# Execute all commands that currently work
+echo -n "Chef Inspec "
+inspec -v
+echo "Chef Infra"
+echo -n " - knife "
+knife -v | awk {'print $NF'}
+echo -n " - chef-client "
+chef-client -v | awk {'print $NF'}
+echo -n " - chef-solo "
+chef-solo -v | awk {'print $NF'}
+echo -n " - chef-apply "
+chef-apply -v | awk {'print $NF'}
+echo -n " - chef-shell "
+chef-shell -v | awk {'print $NF'}
+# TODO(afiune) do we still use this? because it doesn't work
+echo " - chef-zero (not-working)"
```

Comment:
I think we do still need this. Maybe `chef-zero` got munched in the shuffling of what gem provides executables post-license change?

Reply:
It is there, it just doesn't work. I'll fix it on further PRs